documentation/objects/objects.md

```diff
@@ -17,47 +17,38 @@ weight: 1
 
 As the name suggests, the object classification workflow aims to classify full *objects*, based on object-level features and user annotations.
 An *object* in this context is a set of pixels that belong to the same instance.
-In order to do so, the workflow needs *segmentation* images besides the usual raw image data, that can e.g. be generated with the [Pixel Classification Workflow].
-Depending on the availability of these segmentation images, the user can choose between three flavors of object classification workflow, which differ by their input data:
+In order to do so, the workflow needs *segmentation* images besides the usual raw image data, which classify the pixels as either belonging to an object or not. These can be obtained e.g. using the [Pixel Classification Workflow].
+Ilastik has separate workflows depending on the segmentation image flavor:
 
-* Pixel Classification + Object Classification
 * Object Classification [Inputs: Raw Data, Pixel Prediction Map]
 * Object Classification [Inputs: Raw Data, Segmentation]
 
+The combined "Pixel Classification + Object Classification" workflow (found under "Other Workflows") is primarily intended for demonstration purposes and its use in real projects is discouraged. We instead recommend to use the two workflows separately, exporting probability maps from the Pixel Classification workflow and using them as input for the Object Classification workflow.
```

Comment:
matter of taste, but I try to keep one sentence per line in a file. This makes it a little bit easier to track changes (I know about `--word-diff`).